modules/end-user-guide/pages/mounting-volumes.adoc

```diff
@@ -0,0 +1,66 @@
+:navtitle: Mounting Volumes
+:keywords: user-guide, configuring, user, volumes, persistent, volume, claim, mounting, mount
+:page-aliases:
+
+[id="mounting-volumes_{context}"]
+= Mounting Volumes
+
+You can mount a Persistent Volume Claim (PVC) to your {devworkspace} containers. A PVC is used to request persistent storage resources which are abstracted from the user. This section describes how to automatically mount an existing PVC to all {devworkspace} containers.
```

Comment:
We should briefly explain why a user would need to mount a PV. Roughly: 

Che provisions containerized development environments (workspaces) and, without persistent volumes, every change would be lost when the the workspace is stopped. 

To avoid losing changes a user can either:
- pre-create a PVC with the right labels (this section)
- specify a PV in a Devfile that will be managed by Che (link to the devfile section). 

Users will want to use Devfile defined volumes when the persisted data is mandatory for development and don't need to be shared across multiple workspaces. Instead users should favor pre-created PVs if not all the developers of a project will need the PV, or when the PV lifecycle goes beyond the lifecycle of a single workspace or finally when the data included in the PV is shared across workspaces. 

Folder `/projects/` of a workspace is mounted as Devfile defined PV by default.

Pre-created PVs are mounted even if the workspace is run in ephemeral mode (the devfile uses the attribute `controller.devfile.io/storage-type: ephemeral`).

**Note**: Persistent volumes come at a cost: given the attaching a persistent volume will make workspaces startup slower.

Reply:
The other thing I am wondering: what happens when you have one pre-created RWO PV and you start 2 workspaces concurrently? The seconde workspace should not start. That's a big limitation that should be mentioned.

Reply:
> The other thing I am wondering: what happens when you have one pre-created RWO PV and you start 2 workspaces concurrently? 

Hi @amisevsk , do you know the behaviour in this case by any chance?

Reply:
If the cluster enforces RWO, then the second workspace will fail to start, likely with a failedMount error message referencing the PVC. This limitation could be mentioned, but Che dashboard is intended to enforce a limit of one running workspace at a time, so it may not be necessary or relevant.

Reply:
@amisevsk dashboard limits users to have one workspace at a time when the PVC stragegy is common and when the storage type is not `ephemeral`. And this is the default. But if the strategy is `per-workspace` rather than `common` and if the storage type is `ephemeral` then the dashboard should not limit the user to run one workspace at a time.

That said I have tested creating a PVC with the automount labels and starting 2 ephemeral DevWorkspaces and it worked 😮 !

<img width="1790" alt="image" src="https://user-images.githubusercontent.com/606959/165333866-1e7d6c05-9a02-4704-8dfc-a6340b9a22a3.png">

That's because I was lucky and the 2 DW got provisioned on the same node. From the [doc](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-mode):

> **ReadWriteOnce**
   The volume can be mounted as read-write by a single node. ReadWriteOnce access mode still can allow multiple pods to access the volume when the pods are running on the same node.

Reply:
Ah I forgot that limitation was only for `common` workspaces. In that case, it's definitely worth documenting the restrictions as explained by Mario

Reply:
Thank you everyone, I've added this warning:
![image](https://user-images.githubusercontent.com/83611742/165401276-15481d7f-6cd4-4982-bbf9-6d9cadb1ad12.png)